docs/pages/zero-trust-access/deploy-a-cluster/aws-kms.mdx

```diff
@@ -84,12 +84,14 @@ Start by creating the following AWS IAM policy in your account.
                 "kms:ScheduleKeyDeletion",
                 "kms:DescribeKey",
                 "kms:ListResourceTags",
-                "kms:Sign"
+                "kms:Sign",
+                "kms:Decrypt",
             ],
             "Resource": "*",
             "Condition": {
                 "StringEquals": {
-                    "aws:ResourceTag/TeleportCluster": "<Var name="teleport.example.com" description="Your Teleport cluster name"/>"
+                    "aws:ResourceTag/TeleportCluster": "<Var name="teleport.example.com" description="Your Teleport cluster name"/>",
+                    "aws:ResourceTag/TeleportClusterEncryption": "<Var name="teleport.example.com" description="Your Teleport cluster name"/>"
```

Comment:
I also think this is going to require the key to have both the tags rather than one or the other